test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/OtlpExporterOptionsExtensionsTests.cs

```diff
@@ -157,4 +160,88 @@ public void AppendPathIfNotPresent_TracesPath_AppendsCorrectly(string inputUri,
 
         Assert.Equal(expectedUri, resultUri.AbsoluteUri);
     }
+
+    [Theory]
+    [InlineData(OtlpExportProtocol.Grpc, typeof(OtlpGrpcTraceExportClient), false)]
+    [InlineData(OtlpExportProtocol.HttpProtobuf, typeof(OtlpHttpTraceExportClient), false)]
+    [InlineData(OtlpExportProtocol.HttpProtobuf, typeof(OtlpHttpTraceExportClient), true)]
+    public void GetTraceTransmissionHandler_InitializesCorrectExportClientAndTimeoutValue(OtlpExportProtocol protocol, Type exportClientType, bool customHttpClient)
+    {
```

Comment:
I think we could just use one test to check the transmission handler for all three signals.